frontend/src/App.tsx

```diff
@@ -10,6 +10,11 @@ import Default from "./components/pages/Default";
 import DisplayPage from "./components/pages/DisplayPage";
 import NotFound from "./components/pages/NotFound";
 import UpdatePage from "./components/pages/UpdatePage";
+import AdminDasboard from "./components/pages/admin/AdminDashboard";
+import CourseDashboard from "./components/pages/course/CourseDashboard";
+import CourseEditor from "./components/pages/course/CourseEditor";
+import CourseEditorOverview from "./components/pages/course/CourseEditorOverview";
+import CourseOverview from "./components/pages/course/CourseOverview";
```

Comment:
Please rename the Course editor components to Module editors. As well, I don't think CourseEditorOverview page - see comment on lines 102-106 for reasoning.

Reply:
While I am doing this right now, I think we'd still need a course editor component since it seems like we do directly edit courses? I'd think we'd need a module edit page in addition to this.